Commands/ShowRecipeCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using WPF_Restaurant.Models;
+using WPF_Restaurant.Stores;
+using WPF_Restaurant.ViewModels;
+using WPF_Restaurant.Views;
+
+namespace WPF_Restaurant.Commands
+{
+    public class ShowRecipeCommand : BaseCommand
+    {
+        private MainChefViewModel _mainChefViewModel;
+        private readonly Restaurant _restaurant;
+
+        public ShowRecipeCommand(MainChefViewModel mainChefViewModel, Restaurant restaurant)
+        {
+            _mainChefViewModel = mainChefViewModel;
+            _restaurant = restaurant;
+        }
+
+        public override void Execute(object? parameter)
```

Comment:
Why we send array here? Maybe we can send whole ViewModel, see other comment start with "If we bind whole ViewModel"